Update issue templates (#17)
camaraproject/Marketing: .github/ISSUE_TEMPLATE/issue_pm_template.md

Reviewer: Please keep this file ... just replace "Subproject" by "Repository". It would apply e.g. for an issue for this PR here or changes of codeowner ship etc.

Reviewer: Does marketing material also includes the website?

> ---
> name: ☁ Governance management 🎂
> about: Indicate an issue with the project management of CAMARA
> title: ''
> labels: 'project management'
> assignees: ''
> 
> ---

Reviewer: I suggest that the Marketing working group is not responsible for Governance issues ... here the typical repository management issue:

```suggestion
---
name: ☁ Repository management 🎂
about: Indicate an issue with repository or release management
title: ''
labels: 'repository management'
assignees: ''
---
```
